ui/app/js/controllers/rule_scheduled_change_new_controller.js

```diff
@@ -66,6 +66,12 @@ function($scope, $http, $modalInstance, CSRF, Releases, Rules, scheduled_changes
         $scope.sc.sc_data_version = 1;
         $scope.sc.sc_id = response.sc_id;
         $scope.scheduled_changes.push($scope.sc);
+        $scope.sc.scheduled_by = Rules.getScheduledByUser()
+          .then(function successCallback(response) {
+            return response.data.username;
+          },function errorCallback(response) {
+            console.log(response.data);
+          });
```

Comment:
I think the problem here is that you've made $scope.sc.scheduled_by a reference to the Rules.getScheduledByUser() function. The return from the successCallback doesn't get assigned to $scope.sc.scheduled_by - it goes nowhere. Try assigning to $scope.sc.scheduled_by within the successCallback instead.